### PR TITLE
Made AI stable

### PR DESCRIPTION
**Pull request by VibrantJack**
The AI won't just crash the game.
Full games can now be run with AI.
Testing usually does AI vs AI.
Just remove one of the AI objects from testing grouns and fill in UnitTest::test() with what you need to spawn for testing none AI.
If you need only 1, drop 1 of the controllers, if you need none, drop both of them.

### CODE REVIEW

File: _Project/data/scene/testinggrounds.json [hunk 1]
```diff
@@ -61,6 +65,6 @@
 	}
   ],
 	"scenes" : [
-		"Game_Scene.json", "mountainForest.json"
+		"Game_Scene.json"
```
Comment (calmackenzie) [inline _Project/data/scene/testinggrounds.json]: can't commit removal of background to master

File: ability/ability/Ability.cpp [hunk 2]
```diff
@@ -194,7 +194,6 @@ std::vector<kitten::K_GameObject*> ability::Ability::getCardsInHand()
 void ability::Ability::triggerTPEvent(ability::TimePointEvent::TPEventType p_tp, unit::Unit * p_target, AbilityInfoPackage * p_info)
 {
 	unit::StatusContainer* sc = p_target->getStatusContainer();
-	if (sc == nullptr) return;
```
Comment (calmackenzie) [inline ability/ability/Ability.cpp]: Is this change intentional? Not sure what the effects on abilities are

Comment (VibrantJack) [reply ability/ability/Ability.cpp]: O, donno, it sometimes crashed when it was going through because that was null. 
put the check and forgot about it, dont know if that would cause problems or fix any.

Comment (calmackenzie) [reply ability/ability/Ability.cpp]: because what was null ?

Comment (VibrantJack) [reply ability/ability/Ability.cpp]: oh this update removes it. 
My bad, I put the sc there in the original AI code, i don't know why this one removes it.
